packages/core/src/Stores/notification-store.js

```diff
@@ -20,7 +20,7 @@ import { BinaryLink } from 'App/Components/Routes';
 import { action, computed, observable, reaction } from 'mobx';
 import React from 'react';
 import { WS } from 'Services';
-import { sortNotifications, sortNotificationsMobile } from '../App/Components/Elements/NotificationMessage/constants';
+import { sortNotifications, sortNotificationsMobile } from '../App/Components/Elements/NotificationMessage';
```

Comment:
Why is this change needed?

Reply:
Unnecessary path for exported files. It was just proposed by IDE. 
Will return it back